src/components/calculator.js

```diff
@@ -10,6 +11,7 @@ const CalculatorApp = () => {
   return (
     <div>
       <div className="container-div">
+        <Quotes />
```

Comment:
- [ ] Great job working on the `Quotes` component. :+1: Let's do a bit of refactoring to achieve the project requirements. According to the requirement, we are supposed to import the `Quotes` component into the main component. Kindly move it to the main component for a better structure. 🤝️

**Project requirement**
![Screenshot from 2023-03-23 09-55-39](https://user-images.githubusercontent.com/17243325/227129753-9b3091fe-4a29-4eb8-b12d-dd2f982dd2aa.png)

Reply:
This change has been implemented well